src/components/LegendWrapper/LegendWrapper.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { ThemeProvider } from 'styled-components';
+
+import { su2cTheme, Box } from '..';
+import Radio from '../Radio';
+import CheckBox from '../Checkbox';
+
+import LegendWrapper, { LegendWrapperProps } from '.';
+
+export default {
+  title: 'Legend Wrapper',
+  component: LegendWrapper,
+} as Meta<LegendWrapperProps>;
+
+const Template: Story<LegendWrapperProps> = args => <LegendWrapper {...args} />;
+
+export const LegendWrapperDefault: Story<LegendWrapperProps> = Template.bind({});
+LegendWrapperDefault.storyName = 'LegendWrapper';
+LegendWrapperDefault.args = {
+  legend: 'Legend Example',
+};
+
+const TemplateWithRadio: Story<LegendWrapperProps> = args => (
+  <LegendWrapper {...args}>
+    <Box marginBottom="none">
+      <Radio name="example1" value="one">
+        Option one
+      </Radio>
+    </Box>

```

Comment:
mutliple radios and checkboxes in the examples would show wether or not we got the spacing correct